utilities/physics_tendency_updaters.meta

```diff
@@ -0,0 +1,303 @@
+  intent = in
+[ u ]
+  standard_name = x_wind
+  units = m s-1
+  type = real | kind = kind_phys
+  dimensions = (horizontal_loop_extent, vertical_layer_dimension)
+  intent = inout
+[ dudt ]
+  standard_name = tendency_of_x_wind_due_to_model_physics
+  units = m s-2
+  type = real | kind = kind_phys
+  dimensions = (horizontal_loop_extent, vertical_layer_dimension)
+  intent = in
+[ dt ]
+  standard_name = timestep_for_physics
+  long_name = time step
+  units = s
+  dimensions = ()
+  type = real | kind = kind_phys
+  intent = in
+[ errcode ]
+  standard_name = ccpp_error_code
+  long_name = Error flag for error handling in CCPP
+  units = flag
+  type = integer
+  dimensions = ()
+  intent = out
+[ errmsg ]
+  standard_name = ccpp_error_message
+  long_name = Error message for error handling in CCPP
+  units = 1
+  type = character | kind = len=512
+  dimensions = ()
+  intent = out
+#####################################################################
+[ccpp-table-properties]
+  name = apply_tendency_of_y_wind
+  type = scheme
+[ccpp-arg-table]
+  name = apply_tendency_of_y_wind_run
+  type = scheme
+[ nz ]
+  standard_name = vertical_layer_dimension
+  long_name = number of columns
+  units = count
+  type = integer
+  dimensions = ()
+  intent = in
+[ dv ]
+  standard_name = tendency_of_y_wind
+  units = m s-2
+  type = real | kind = kind_phys
+  dimensions = (horizontal_loop_extent, vertical_layer_dimension)
+  intent = in
+[ v ]
+  standard_name = y_wind
+  units = m s-1
+  type = real | kind = kind_phys
+  dimensions = (horizontal_loop_extent, vertical_layer_dimension)
+  intent = inout
+[ dvdt ]
+  standard_name = tendency_of_y_wind_due_to_model_physics
+  units = m s-2
+  type = real | kind = kind_phys
+  dimensions = (horizontal_loop_extent, vertical_layer_dimension)
+  intent = in
+[ dt ]
+  standard_name = timestep_for_physics
+  long_name = time step
+  units = s
+  dimensions = ()
+  type = real | kind = kind_phys
+  intent = in
+[ errcode ]
+  standard_name = ccpp_error_code
+  long_name = Error flag for error handling in CCPP
+  units = flag
+  type = integer
+  dimensions = ()
+  intent = out
+[ errmsg ]
+  standard_name = ccpp_error_message
+  long_name = Error message for error handling in CCPP
+  units = 1
+  type = character | kind = len=512
+  dimensions = ()
+  intent = out
+#####################################################################
+[ccpp-table-properties]
+  name = apply_heating_rate
+  type = scheme
+[ccpp-arg-table]
+  name = apply_heating_rate_run
+  type = scheme
+[ nz ]
+  standard_name = vertical_layer_dimension
+  long_name = number of columns
+  units = count
+  type = integer
+  dimensions = ()
+  intent = in
+[ s ]
+  standard_name = heating_rate
+  units = J kg-1 s-1
+  type = real | kind = kind_phys
+  dimensions = (horizontal_loop_extent, vertical_layer_dimension)
+  intent = in
+[ temp ]
+  standard_name = air_temperature
+  state_variable = true
```

Comment:
I'm still unclear about this metadata field.  Should the other "state" variables have it (like y_wind)?

Reply:
I will adjust these. I think the rule should be _is there a tendency for this variable?_. So `x_wind` and `y_wind` are in the club but not `geopotential_height`.